### PR TITLE
Added endpoint for `yank` and `unyank` cargo commands

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #15 
Added slice to handle `yank` and `unyank` cargo commands.

### CODE REVIEW

File: src/test/java/com/artipie/cargo/http/YankSliceTest.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+     */
+    private Storage asto;
+
+    @BeforeEach
+    void init() {
+        this.asto = new InMemoryStorage();
+    }
+
+    @Test
+    void yankesMetadataAndReturnsOk() {
```
Comment (genryxy) [inline src/test/java/com/artipie/cargo/http/YankSliceTest.java:45]: I'd suggest to add test case for `unyank` operation

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one comment

File: src/main/java/com/artipie/cargo/http/YankSlice.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import java.util.regex.Pattern;
+import javax.json.Json;
+import org.reactivestreams.Publisher;
+
+/**
+ * Slice to process `yank` and `unyank` cargo requests.<br/>
+ * See<br/>
+ * https://doc.rust-lang.org/cargo/reference/registries.html#yank<br/>
+ * https://doc.rust-lang.org/cargo/reference/registries.html#unyank
+ * @since 0.1
```
Comment (genryxy) [inline src/main/java/com/artipie/cargo/http/YankSlice.java:28]: @olenagerasimova and according to docs `unyank` operation has `PUT` method

**Comment by olenagerasimova** [comment]
@genryxy thanks, all corrected

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, looks good to me

**Pull request merged**